tests/test_unit/test_transfers/test_delete_transfer.py

```diff
@@ -16,9 +17,8 @@ async def test_maintainer_plus_can_delete_group_transfer(
 ):
     # Arrange
     user = group_transfer.owner_group.get_member_of_role(role_maintainer_plus)
-    transfer = await session.get(Transfer, group_transfer.id)
-
-    assert not transfer.is_deleted
+    t_id = group_transfer.id
+    await session.get(Transfer, group_transfer.id)
```

Comment:
According to [Session.get(...)](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.get), it may return None